x/axelarnet/keeper/msg_server.go

```diff
@@ -350,11 +354,7 @@ func (s msgServer) RouteIBCTransfers(c context.Context, _ *types.RouteIBCTransfe
 				continue
 			}
 
-			err = s.EnqueueTransfer(ctx, types.NewIBCTransfer(sender, p.Recipient.Address, token, portID, channelID))
-			if err != nil {
-				return nil, err
-			}
-
+			funcs.MustNoErr(s.EnqueueTransfer(ctx, types.NewIBCTransfer(sender, p.Recipient.Address, token, portID, channelID)))
```

Comment:
Reuse the transferID from the nexus module that's present in the pendingTransfer